engines/native/sandbox.go

```diff
@@ -30,6 +30,25 @@ type sandbox struct {
 	wg         atomics.WaitGroup
 }
 
+// white list of environment variables that must
+// be passed to the child process
+var environmentWhitelist = []string{
+	"PATH",
+	"TMPDIR",
+	"PWD",
+	"EDITOR",
+	"LANG",
+	"LOGNAME",
+	"TERM",
+	"TERM_PROGRAM",
+	"TASK_ID",
```

Comment:
Shouldn't these be set based on config variables rather than being passed through